app/src/main/java/io/apicurio/registry/content/ContentHandleMessageBodyWriter.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.content;
+
+import io.apicurio.registry.utils.IoUtil;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.MessageBodyWriter;
+import javax.ws.rs.ext.Provider;
+
+/**
+ * @author Ales Justin
+ */
+@Provider
+@Produces(MediaType.WILDCARD)
+public class ContentHandleMessageBodyWriter implements MessageBodyWriter<ContentHandle> {
+    @Override
+    public boolean isWriteable(Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType) {
+        if (type == StringContentHandle.class) {
+            return (mediaType.equals(MediaType.APPLICATION_JSON_TYPE) == false);
+        }
```

Comment:
What is the purpose of this?  If the type is a `StringContentHandle`, why return `true` only when the type is something other than application/json?  Won't the media type be application/json quite often?  Pretty much for everything other than protobuf?

Reply:
This class here is to handle things really as stream; e.g. jars, archives, etc -- when we fully support this.
Atm I left a wildcard, which I guess should be changed?
If we do this, then this check can be removed.
I left it so any current string content + json is not handled by this wildcard ...